src/main.ts

```diff
@@ -51,24 +50,21 @@ async function start () {
 
   // initialize Launch Darkly
   if (window['ldClientId']) {
-    await initLDClient()
+    await initLdClient()
   }
 
-  // check app feature flag
-  if (featureFlags.getFlag('bcrs-create-ui-enabled')) {
```

Comment:
Now that we have been live for a few months, this FF is obsolete, so I removed it.